appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/manage-linked-appeals.router.js

```diff
@@ -4,9 +4,22 @@ import * as validators from './manage-linked-appeals.validators.js';
 
 const router = createRouter({ mergeParams: true });
 
-router.route('/linked-appeals').get(controller.getLinkedAppeals);
+router
+	.route('/add')
+	.get(controller.getAddLinkedAppealReference)
+	.post(validators.validateAddLinkedAppealReference, controller.postAddLinkedAppeal);
+
+router
+	.route('/add/check-and-confirm')
+	.get(controller.getAddLinkedAppealCheckAndConfirm)
+	.post(
+		validators.validateAddLinkedAppealCheckAndConfirm,
+		controller.postAddLinkedAppealCheckAndConfirm
```

Comment:
We should get all the controller actions wrapped in the async component. There are a few that are missing it, may be worth not adding more? Or leave it and we'll run it through for this ticket: https://pins-ds.atlassian.net/browse/BOAT-600

Reply:
Thanks, I've updated the routes I added to use this wrapper now